src/main/kotlin/model/SnowballRException.kt

```diff
@@ -114,4 +114,13 @@ sealed class SnowballRException(
             entityType: String,
         ) : InvalidIdException(id, entityType, "UUID")
     }
+
+    /**
+     * Represents a specific type of exception that occurs when a call has the wrong preconditions.
+     *
+     * @constructor Creates an [FailedPreconditionException] with the id of the user.
```

Comment:
```suggestion
     * @constructor Creates a [FailedPreconditionException] with the id of the user.
```
Also the description is not quite correct.

Reply:
Good catch.